xml/vk.xml

```diff
@@ -5993,7 +5993,7 @@ typedef void <name>CAMetalLayer</name>;
         </type>
         <type category="struct" name="VkPhysicalDeviceRayTracingMotionBlurFeaturesNV" structextends="VkPhysicalDeviceFeatures2,VkDeviceCreateInfo">
             <member values="VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_RAY_TRACING_MOTION_BLUR_FEATURES_NV"><type>VkStructureType</type> <name>sType</name></member>
-            <member noautovalidity="true"><type>void</type>*        <name>pNext</name></member>
+            <member optional="true" noautovalidity="true"><type>void</type>*        <name>pNext</name></member>
```

Comment:
I'm not sure what the interaction is going to be like here - we don't usually have optional and notautovalidity on the same parameter - @oddhack  any idea what this might do?

Reply:
Nothing, AFAICT. There are two structures with optional & noautovalidity both "true" in vk.xml - VkCopyCommandTransformInfoQCOM and VkPhysicalDeviceShaderTerminateInvocationFeaturesKHR - and removing the optional="true" has zero effect on generated VUs.

I think the only way to deal with this going forward will be adding a CI check to require optional="true" on pNext members. It is a general problem with people want to infer things from the XML that are not relevant to any of the Khronos uses of it. Fortunately this is a fairly straightforward check.

Reply:
Is this CI check a blocker for this PR?

Reply:
No, I'll track it separately. We have some XML sanity checking tools which aren't currently enabled but can be soon, and this is a sensible check to add when we do that.

Reply:
@ashpil check out #1597, which (hopefully) implements the check. It would be helpful if you could cherry-pick those changes (or rebase on / retarget that branch) and run scripts/xml_consistency.py, or just enable Github Actions on your repository so our CI will run. I am fairly sure there are at least a few more pNexts your tool didn't catch.

https://github.com/KhronosGroup/Vulkan-Docs/pull/1597/checks?check_run_id=3241250501

Reply:
@oddhack Done. The script was using `is not` when it should've been using `!=`, so it was giving quite a few false positives. 

I fixed that, and one warning remains about `vkGetDrmDisplayEXT` - not sure what that is about. Also `spec-generate` seems to error.

Reply:
vkGetDrmDisplayEXT violated our own naming conventions. There's no way to fix it in the C API at this point, so it is treated as an exception (thus the warning instead of an error that would break CI).

The other problem seems to derive from my updating the Docker image used for CI to add a new component needed by the build. Unfortunately it also updated nodejs from 10.x to 12.x and that is breaking an unrelated postprocessing step used on the generated HTML (via scripts/translate_math.js). I frankly don't know enough about the issue, which is something to do with ES6 import vs. CommonJS require vs. transpilation vs. package.json vs. bundling vs. about 5 other things I've found references to in a short time searching, to understand what can be done to fix this, so I will probably have to figure out how to back the image off to the older nodejs for the time being.

Reply:
OK, resolved now. It was using a more recent version of escape-string-regexp that was apparently an ES6 module, locking the container build to the previously used version is a workaround.